### PR TITLE
pySurfex EnKF

### PR DESCRIPTION
**Pull request by josteinblyverket**
Latest updates to allow for the EnKF in HARMONIE-AROME

### CODE REVIEW

**Comment by trygveasp** [comment]
Why do you turn off snow observations? What about snow assimilation?

**Comment by josteinblyverket** [comment]
Updated with metno/pysurfex master merge.

File: examples/cfg/config_exp_EnKF.toml [hunk 1]
```diff
@@ -125,15 +125,15 @@ CFILE_FORMAT_LSM = "ASCII"              # File format for land sea mask when usi
 [SURFEX.ASSIM.SCHEMES]
 SEA = "NONE"                         # Assimilation method for sea tile: "INPUT"|"NONE"
 INLAND_WATER = "INPUT"               # Assimilation method for water tile: "INPUT"|"NONE"
-ISBA = "ENKF"                         # Assimilation method for ISBA soil: "NONE"|"OI"|"EKF"
+ISBA = "ENSRKF"                         # Assimilation method for ISBA soil: "NONE"|"OI"|"EKF"
```
Comment (trygveasp) [inline examples/cfg/config_exp_EnKF.toml:128]: You must change this

File: examples/cfg/config_exp_EnKF.toml [hunk 2]
```diff
@@ -125,15 +125,15 @@ CFILE_FORMAT_LSM = "ASCII"              # File format for land sea mask when usi
...
 TEB = "NONE"                         # Assimilation method for TEB tile: "NONE"|"ROADT"
 
 [SURFEX.ASSIM.OBS]
 LOBSHEADER = false                          # ASCII observation file has header
 CFILE_FORMAT_OBS = "ASCII"                  # File format for observations when using SODA: "ASCII"|"FA"
 LOBSNAT = false                             # Observation ASCII file only contains nature points
 COBS_M = ["T2M","HU2M","WG2 ","LAI","SWE"]  # Observation types
-NNCO = [1, 1, 0, 0, 1]                      # Active observation types
+NNCO = [1, 1, 0, 0, 0]                      # Active observation types
```
Comment (trygveasp) [inline examples/cfg/config_exp_EnKF.toml:136]: And change this

**Review by trygveasp — CHANGES_REQUESTED** [review]

**Pull request closed without merging**